### PR TITLE
Add referrer dialog

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/LikeRegisterForm.vue [hunk 1]
```diff
@@ -279,8 +292,30 @@ export default {
         }
       }
     },
+    async fetchReferrerInfo() {
+      this.shouldShowReferrerDialog = true;
+      this.setErrorDisabled(true);
+      try {
+        const response = await api.apiGetUserMinById(this.referrer);
```
Comment (williamchong007) [inline components/LikeRegisterForm.vue]: probably want to use a action that call 
```
const { avatar, displayName } = await apiWrapper(ctx, api.apiGetUserMinById(r.id), { slient: true });
```
this kind of style is when apiWrapper didnt support `slient`

File: components/LikeRegisterForm.vue [hunk 2]
```diff
@@ -279,8 +292,30 @@ export default {
         }
       }
     },
+    async fetchReferrerInfo() {
+      this.shouldShowReferrerDialog = true;
```
Comment (williamchong007) [inline components/LikeRegisterForm.vue]: 未 load 到 referrer 嘅 data 嘅時候個 dialog 係咩樣的？

Comment (nwingt) [reply components/LikeRegisterForm.vue]: Updated

File: components/LikeRegisterForm.vue (hunk 1 above)
Comment (nwingt) [reply components/LikeRegisterForm.vue]: Updated

**Pull request merged**